packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -296,10 +298,12 @@ export class CdkToolkit {
     // --------------                --------  'cdk deploy' done  --------------  'cdk deploy' done  --------------
     let latch: 'pre-ready' | 'open' | 'deploying' | 'queued' = 'pre-ready';
 
+    const logMonitor = options.logs ? new CloudWatchLogEventMonitor({ hotswapTime: new Date() }) : undefined;
```

Comment:
This is the same as the default for `hotswapTime`, no?

Reply:
Yeah that's true, updated to just let the default be sets.